spec/components/placement/summary_component_spec.rb

```diff
@@ -33,6 +33,10 @@
     it "renders the details of each placement" do
       render_inline(component)
 
+      # Availability details
+      expect(page).to have_content("Available", count: 1)
+      expect(page).to have_content("Unavailable", count: 1)
+
```

Comment:
While this technically covers the functionality we've added, I don't think the `have_content` assertion is all that valuable.

Presumably we could flip the label logic around and this test would still pass? Since there would still be one of each.

Maybe it would be better covered within the single context block as two separate scenarios?

https://github.com/DFE-Digital/itt-mentor-services/blob/cfe8b134ed9025ced6b2e4b22effbd1732e93299/spec/components/placement/summary_component_spec.rb#L59